tests/unit/support/decorators.ts

```diff
@@ -54,10 +54,12 @@ registerSuite('decorators', {
 			}
 
 			assert.throws(() => {
+				/* tslint:disable */
```

Comment:
Ignoring linting errors here as the `Target` class is created but never used... but we don't need to use it 😛